app/src/main/java/app/revanced/manager/ui/component/bundle/ImportBundleDialog.kt

```diff
@@ -35,18 +36,19 @@ import app.revanced.manager.util.JAR_MIMETYPE
 fun ImportBundleDialog(
```

Comment:
Shouldn't this also use the bundle type dialog?

Reply:
What do you mean? integrate the BundleTypeDialog into the ImportBundleDialog directly?

Reply:
This composable switches type when you click on the bundle type list item. I think the item should be removed or open the BundleTypeDialog instead of switching type immediately

Reply:
Ah, now I understand, the reviewed line really didn't get the point across

Reply:
https://github.com/ReVanced/revanced-manager/pull/1490#issuecomment-1849069638 considering this, which is out of scope, I'll be marking this as resolved considering that the changes would be immediately deleted in a follow-up commit